README.md

```diff
@@ -8,46 +8,58 @@ Codebase for Sparkle, brought to you by Sparkle.
 
 Clone the repo and cd into it
 
-```
+```bash
 git clone https://github.com/sparkletown/sparkle
 cd sparkle
 ```
 
-Now, obtain the `.env` file for the environemnt (eg. staging) and save it (eg. `.env.staging`)
+<!-- Now, obtain the `.env` file for the environemnt (eg. staging) and save it (eg. `.env.staging`) -->
```

Comment:
No need to comment out these bits, can just delete them.

(only commenting on the first, but this applies to all of the same pattern of issue)